### PR TITLE
MBS-8146 Add filter changing via cli documentation

### PR DESCRIPTION
**Pull request by sergioboi**
add how change filter by name
add how change filter configuration

### CODE REVIEW

File: docs/content/docs/test/Runner.md [hunk 1]
```diff
@@ -187,8 +187,70 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
...
 
+### Change applied filter without changing `build.gradle.kts`
+
+1. Add custom gradle property for filter name to gradle.properties file
+
+```properties
+filterName="filterName"
+```
+
+2. Let plugin configuration depends on `property`
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]:
```suggestion
2. Use the property to configure a plugin
```

- We use backticks only for [exact names](https://avito-tech.github.io/avito-android/docs/contributing/docs/#%D0%B2%D1%8B%D0%B4%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5-%D1%82%D0%B5%D1%80%D0%BC%D0%B8%D0%BD%D0%BE%D0%B2)
- "depends on" can be misinterpreted as grade `dependsOn`

File: docs/content/docs/test/Runner.md [hunk 2]
```diff
@@ -187,8 +187,70 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
...
+
+extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
+    configurationsContainer.register("Local") {
+        filter = filterName
+        // else...   
+    }    
+}
+```
+
+3. Add `property` to cli command if you want to override `filterName`
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]:
```suggestion
3. Add `property` to CLI command if you want to override `filterName`
```

File: docs/content/docs/test/Runner.md [hunk 3]
```diff
@@ -187,8 +187,70 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
...
+```shell script
+./gradlew instrumentationLocal -PincludedAnnotation="package.AnotherAnnotationClassName"
+```
+
 ## Choosing target for tests execution
 
+### Run test on APK was built before
+
+Plugin builds APKs on his own by default.\
+If for any reason you have to build APK externally, you can pass files manually:
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]: Why do we have ":" at the end?

File: docs/content/docs/test/Runner.md [hunk 4]
```diff
@@ -187,8 +187,70 @@ extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
...
+}
+```
+
+3. Add `property` to cli command if you want to override `filterName`
+
+```shell script
+./gradlew instrumentationLocal -PfilterName=<any name of defined filter>
+```
+
+### Customize filter without changing `build.gradle.kts`
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md]:
```suggestion
### Customize a filter without changing `build.gradle.kts`
```

**Review by eugene-krivobokov — APPROVED** [review]

File: docs/content/docs/test/Runner.md (hunk 3 above)
Comment (sergioboi) [reply docs/content/docs/test/Runner.md]: bad merge

**Pull request merged**